detection-rules/link_outlook_left_to_right_exploit.yml

```diff
@@ -9,7 +9,13 @@ source: |
   type.inbound 
   // this regex contains a Unicode U+200E character (may not be visible in many clients)
   and regex.contains(body.html.raw, 'base.{0,100}‎/>')
-tags: 
-  - "Suspicious link"
-  - "Office exploit"
+attack_type:
+  - "Credential Phishing"
+tactics_and_techniques:
+  - "Evasion"
+  - "Exploit"
```

Comment:
```suggestion
```

Reply:
description: |
  The exploit involves tricking Outlook for Windows into displaying a fake domain while opening another one.